lib/builder/webpack/style-loader.js

```diff
@@ -45,15 +45,16 @@ module.exports = function styleLoader(ext, loaders = [], isVueLoader = false) {
     }
   }
 
-  if (this.options.build.extractCSS && !this.options.dev) {
-    return ExtractTextPlugin.extract({
+  if (this.options.build.extractCSS) {
+    const extraction = ExtractTextPlugin.extract({
       fallback: vueStyleLoader,
       use: [
         cssLoader,
         postcssLoader,
         ...loaders
       ].filter(l => l)
     })
+    return this.options.dev ? ['css-hot-loader'].concat(extraction) : extraction
```

Comment:
Won't this conflict with vue-style-loader?

Reply:
`css-hot-loader` is only a wrapper to support HMR in `css`, per my test it should work fine in normal cases. Did I miss anything?

Reply:
If you test it locally that's great :) Let's merge and see it in action ;)

Reply:
Hope everything goes fine 😄